data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/index/ExistingDocumentQueryManager.java

```diff
@@ -168,19 +188,25 @@ private MsearchRequest buildMultiSearchRequest() {
             for (final Map.Entry<String, Map<String, QueryManagerBulkOperation>> entry : bulkOperationsWaitingForQuery.entrySet()) {
                 final String index = entry.getKey();
                 final List<FieldValue> values = getTermValues(entry.getValue().values());
-
-                m.searches(s -> s
-                        .header(h -> h.index(index))
-                        .body(b -> b
-                                .size(values.size())
-                                .source(source -> source.filter(f -> f.includes(queryTerm)))
-                                .query(Query.of(q -> q
-                                        .terms(TermsQuery.of(t -> t
-                                                .field(queryTerm)
-                                                .terms(TermsQueryField.of(tf -> tf.value(values)))
-                                        ))
-                                ))
-                        ));
+                final int batchSize = 1000;
```

Comment:
Do we want to make this configurable?

Reply:
We could make it configurable I suppose.